Sources/MapboxNavigation/NavigationViewControllerDelegate.swift

```diff
@@ -483,35 +523,28 @@ public extension NavigationViewControllerDelegate {
         logUnimplemented(protocolType: NavigationViewControllerDelegate.self, level: .debug)
         return nil
     }
-
-    /**
-     `UnimplementedLogging` prints a warning to standard output the first time this method is called.
-     */
-    func navigationViewController(_ navigationViewController: NavigationViewController, didSelect route: Route) {
```

Comment:
Such method is not actually exposed anywhere, can be removed.

Reply:
I do not think that anybody requested it recently, so we can add it when it's really needed.